README.md

```diff
@@ -11,30 +11,63 @@
 **This is the README for the `2.x` version of Redoc (React-based).** 
 **The README for the `1.x` version is on the [v1.x](https://github.com/Redocly/redoc/tree/v1.x) branch**
 
-![ReDoc demo](https://raw.githubusercontent.com/Redocly/redoc/master/demo/redoc-demo.png)
+## About Redoc
 
-## [Live demo](http://redocly.github.io/redoc/)
+Redoc is an open-source tool for generating documentation from OpenAPI (fka Swagger) definitions. 
 
-[<img alt="Deploy to Github" src="http://i.imgur.com/YZmaqk3.png" height="60px">](https://github.com/Rebilly/generator-openapi-repo#generator-openapi-repo--) [<img alt="ReDoc as a service" src="http://i.imgur.com/edqdCv6.png" height="60px">](https://redoc.ly) [<img alt="Customization services" src="http://i.imgur.com/c4sUF7M.png" height="60px">](https://redoc.ly/#services)
+By default Redoc offers a three-panel, responsive layout: 
+
+- The left panel contains a search bar and navigation menu.
+- The central panel contains the documentation.
+- The right panel contains request and response examples.
+
+![Redoc demo](https://raw.githubusercontent.com/Redocly/redoc/master/demo/redoc-demo.png)
+
+## Live demo
+
+If you want to see how Redoc will render your OpenAPI definition,
+you can try it out online at https://redocly.github.io/redoc/.
+
+A version of the Swagger Petstore API is displayed by default.
+To test it with your own OpenAPI definition,
+enter the URL for your definition and select **TRY IT**.
+
+## Redoc vs. Redocly Reference docs
+
+Redoc Community Edition is free open-source software (OSS).
+Redocly has developed extensive enhancements on top of Redoc Community Edition's open core, creating Redocly Reference Docs.
+See [Comparison Redoc CE vs. Redocly Reference Docs](https://redoc.ly/redoc-vs-reference/)
+for the main differences between the Redoc Community Edition and Redocly Reference Docs.
+
+## Deploy to GitHub
+[<img alt="Deploy to Github" src="http://i.imgur.com/YZmaqk3.png" height="60px">](https://github.com/Rebilly/generator-openapi-repo#generator-openapi-repo--)
```

Comment:
**Thought**: This links to a repo which is outdated. We should think it through more, it has been superseded by:

1. Redocly/OpenAPI-starter
2. Redocly/OpenAPI-cli "split" command

We should probably update that repo. Also, maybe the "Deploy to GitHub" doesn't even make sense any longer, because that script was removed in an older create-OpenAPI-repo version. 

I would like to consult @RomanHotsiy's opinion too.

Side note/thought: I think we definitely need a link to the OpenAPI-cli repo. It powers the core of Redoc now, and it is a great tool for linting OpenAPI docs.

Reply:
Yes, thanks.